subprojects/docs/src/samples/android-application/kotlin/app/build.gradle.kts

```diff
@@ -22,10 +22,10 @@ android {
 }
 
 dependencies {
-    implementation(fileTree(mapOf("dir" to "libs", "include" to listOf("*.jar")))
-            implementation("androidx.appcompat:appcompat:1.0.2")
-            implementation("androidx.constraintlayout:constraintlayout:1.1.3")
-            testImplementation("junit:junit:4.12")
-            androidTestImplementation("androidx.test:runner:1.1.1")
-            androidTestImplementation("androidx.test.espresso:espresso-core:3.1.1")
+    implementation(fileTree(mapOf("dir" to "libs", "include" to listOf("*.jar"))))
```

Comment:
Does this sample actually have libs in this directory?  if not, I think we should remove this.